src/views/tdbmGenerate.php

```diff
@@ -56,8 +56,43 @@
 </div>
 
 <div class="control-group">
+	<label class="control-label">Use the default path (compute with PSR):</label>
+	<div class="controls">
+		<input type="checkbox" id="defaultPath" name="defaultPath" value="1" <?php echo $this->defaultPath ? 'checked="checked"' : '' ?>></input>
+		<span class="help-block">Select this option if you want to use the default folder to store daos and beans.</span>
```

Comment:
Remplace le texte par:

Use this option if you want TDBM to autodetect the source directory where DAOs and beans will be stored from the namespace and `composer.json` autoloader.